apps/desktop/scripts/dev.js

```
@@ -1,16 +1,16 @@
-/* eslint-disable no-plusplus */
 const path = require('path');
-const { exec, execSync, spawnSync } = require('child_process');
+const { execSync } = require('child_process');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)